src/main/java/graphql/annotations/type/GraphQLUndefined.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ */
+package graphql.annotations.type;
+
+import graphql.schema.GraphQLSchemaElement;
+import graphql.schema.GraphQLType;
+import graphql.schema.GraphQLTypeVisitor;
+import graphql.schema.SchemaElementChildrenContainer;
+import graphql.util.TraversalControl;
+import graphql.util.TraverserContext;
+
+import java.util.List;
+
+public class GraphQLUndefined implements GraphQLType {
```

Comment:
I don't understand the meaning or usage of this class. It's only used in buildArg, in a value that will be thrown away just after.